{
    "data-feed-bybit-perpetual-cca5766921-ss-0": {
        "events": [
            "2022-05-25 14:07:26.833400, WAITING_FOR_POD_TO_BE_SCHEDULED, None, None",
            "2022-05-25 14:07:26.914910, PodObjectLoggedEvent.POD_PHASE_CHANGED, {'phase': 'Pending'}, None",
            "2022-05-25 14:07:26.914910, PodObjectLoggedEvent.POD_PHASE_CHANGED, {'phase': 'Pending'}, None",
            "2022-05-25 14:07:26.964746, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'PodScheduled', 'status': 'True'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:26.964746, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'PodScheduled', 'status': 'True'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:26.995160, PodKubeLoggedEvent.POD_EVENT, {'reason': 'Scheduled', 'count': 1, 'message': 'Successfully assigned data-feed/data-feed-bybit-perpetual-cca5766921-ss-0 to minikube-1-m02'}, 2022-05-25 11:07:27+00:00",
            "2022-05-25 14:07:26.995160, PodKubeLoggedEvent.POD_EVENT, {'reason': 'Scheduled', 'count': 1, 'message': 'Successfully assigned data-feed/data-feed-bybit-perpetual-cca5766921-ss-0 to minikube-1-m02'}, 2022-05-25 11:07:27+00:00",
            "2022-05-25 14:07:26.995392, POD_SCHEDULED, None, None",
            "2022-05-25 14:07:26.995454, WAITING_FOR_DF_CONTAINER_TO_PULL_IMAGE, None, None",
            "2022-05-25 14:07:27.106128, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Initialized', 'status': 'True'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:27.106128, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Initialized', 'status': 'True'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:27.106190, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Ready', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:27.106190, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Ready', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:27.106224, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'ContainersReady', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:27.106224, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'ContainersReady', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:27.106247, PodObjectLoggedEvent.CONTAINER_STATE_CHANGED, {'state': {'waiting': {'reason': 'ContainerCreating'}}}, None",
            "2022-05-25 14:07:27.106247, PodObjectLoggedEvent.CONTAINER_STATE_CHANGED, {'state': {'waiting': {'reason': 'ContainerCreating'}}}, None",
            "2022-05-25 14:07:27.106259, PodObjectLoggedEvent.CONTAINER_STARTUP_PROBE_STATE_CHANGED, {'started': False}, None",
            "2022-05-25 14:07:27.106259, PodObjectLoggedEvent.CONTAINER_STARTUP_PROBE_STATE_CHANGED, {'started': False}, None",
            "2022-05-25 14:07:27.106270, PodObjectLoggedEvent.CONTAINER_READINESS_PROBE_STATE_CHANGED, {'ready': False}, None",
            "2022-05-25 14:07:27.106270, PodObjectLoggedEvent.CONTAINER_READINESS_PROBE_STATE_CHANGED, {'ready': False}, None",
            "2022-05-25 14:07:27.106280, PodObjectLoggedEvent.CONTAINER_RESTART_COUNT_CHANGED, {'restartCount': 0}, None",
            "2022-05-25 14:07:27.106280, PodObjectLoggedEvent.CONTAINER_RESTART_COUNT_CHANGED, {'restartCount': 0}, None",
            "2022-05-25 14:07:33.968735, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Pulled', 'count': 1, 'message': 'Container image \"050011372339.dkr.ecr.ap-northeast-1.amazonaws.com/anov/svoe_data_feed:v1.0.3\" already present on machine'}, 2022-05-25 11:07:34+00:00",
            "2022-05-25 14:07:33.968735, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Pulled', 'count': 1, 'message': 'Container image \"050011372339.dkr.ecr.ap-northeast-1.amazonaws.com/anov/svoe_data_feed:v1.0.3\" already present on machine'}, 2022-05-25 11:07:34+00:00",
            "2022-05-25 14:07:33.969207, DF_CONTAINER_IMAGE_PULLED, None, None",
            "2022-05-25 14:07:33.969332, WAITING_FOR_POD_TO_START_ESTIMATION_RUN, None, None",
            "2022-05-25 14:07:34.114157, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Created', 'count': 1, 'message': 'Created container data-feed-container'}, 2022-05-25 11:07:34+00:00",
            "2022-05-25 14:07:34.114157, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Created', 'count': 1, 'message': 'Created container data-feed-container'}, 2022-05-25 11:07:34+00:00",
            "2022-05-25 14:07:34.427536, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Started', 'count': 1, 'message': 'Started container data-feed-container'}, 2022-05-25 11:07:34+00:00",
            "2022-05-25 14:07:34.427536, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Started', 'count': 1, 'message': 'Started container data-feed-container'}, 2022-05-25 11:07:34+00:00",
            "2022-05-25 14:07:35.123491, PodObjectLoggedEvent.POD_PHASE_CHANGED, {'phase': 'Running'}, None",
            "2022-05-25 14:07:35.123491, PodObjectLoggedEvent.POD_PHASE_CHANGED, {'phase': 'Running'}, None",
            "2022-05-25 14:07:35.123695, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Ready', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:35.123695, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Ready', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:35.123794, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'ContainersReady', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:35.123794, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'ContainersReady', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:07:35.123853, PodObjectLoggedEvent.CONTAINER_STATE_CHANGED, {'state': {'running': {'startedAt': '2022-05-25T11:07:34Z'}}}, None",
            "2022-05-25 14:07:35.123853, PodObjectLoggedEvent.CONTAINER_STATE_CHANGED, {'state': {'running': {'startedAt': '2022-05-25T11:07:34Z'}}}, None",
            "2022-05-25 14:07:35.124457, POD_STARTED_ESTIMATION_RUN, None, None",
            "2022-05-25 14:07:35.124535, WAITING_FOR_POD_TO_FINISH_ESTIMATION_RUN, None, None",
            "2022-05-25 14:07:45.128290, POD_FINISHED_ESTIMATION_RUN, None, None",
            "2022-05-25 14:07:45.128352, COLLECTING_METRICS, None, None",
            "2022-05-25 14:07:49.011873, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 1, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:49+00:00",
            "2022-05-25 14:07:49.011873, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 1, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:49+00:00",
            "2022-05-25 14:07:51.006690, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 2, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:51+00:00",
            "2022-05-25 14:07:51.006690, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 2, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:51+00:00",
            "2022-05-25 14:07:53.021678, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 3, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:53+00:00",
            "2022-05-25 14:07:53.021678, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 3, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:53+00:00",
            "2022-05-25 14:07:55.005618, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 4, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:55+00:00",
            "2022-05-25 14:07:55.005618, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'UnhealthyStartup', 'count': 4, 'message': 'Startup probe failed: HTTP probe failed with statuscode: 500'}, 2022-05-25 11:07:55+00:00",
            "2022-05-25 14:07:55.577198, METRICS_COLLECTED_MISSING, None, None",
            "2022-05-25 14:07:55.577268, WAITING_FOR_POD_TO_BE_DELETED, None, None",
            "2022-05-25 14:07:55.843301, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Killing', 'count': 1, 'message': 'Stopping container data-feed-container'}, 2022-05-25 11:07:55+00:00",
            "2022-05-25 14:07:55.843301, PodKubeLoggedEvent.CONTAINER_EVENT, {'reason': 'Killing', 'count': 1, 'message': 'Stopping container data-feed-container'}, 2022-05-25 11:07:55+00:00",
            "2022-05-25 14:08:28.421284, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Ready', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:08:28.421284, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'Ready', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:08:28.421381, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'ContainersReady', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:08:28.421381, PodObjectLoggedEvent.POD_CONDITION_CHANGED, {'type': 'ContainersReady', 'status': 'False', 'reason': 'ContainersNotReady', 'message': 'containers with unready status: [redis redis-exporter data-feed-container]'}, 2022-05-25 11:07:27",
            "2022-05-25 14:08:28.421428, PodObjectLoggedEvent.CONTAINER_STATE_CHANGED, {'state': {'terminated': {'exitCode': 137, 'reason': 'Error', 'startedAt': '2022-05-25T11:07:34Z', 'finishedAt': '2022-05-25T11:08:26Z', 'containerID': 'docker://5e392ed3a08ddf4deaa908b72cca276c5dd668b5e780aa327306c96b865770b3'}}}, None",
            "2022-05-25 14:08:28.421428, PodObjectLoggedEvent.CONTAINER_STATE_CHANGED, {'state': {'terminated': {'exitCode': 137, 'reason': 'Error', 'startedAt': '2022-05-25T11:07:34Z', 'finishedAt': '2022-05-25T11:08:26Z', 'containerID': 'docker://5e392ed3a08ddf4deaa908b72cca276c5dd668b5e780aa327306c96b865770b3'}}}, None",
            "2022-05-25 14:08:28.595758, PodObjectLoggedEvent.POD_DELETED, None, None",
            "2022-05-25 14:08:28.595758, PodObjectLoggedEvent.POD_DELETED, None, None",
            "2022-05-25 14:08:28.595890, POD_DELETED, None, None"
        ],
        "metrics": {
            "health": {
                "health_absent_funding_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "health_absent_l2_book_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "health_absent_liquidations_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "health_absent_open_interest_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "health_absent_trades_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "health_avg_funding_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "health_avg_l2_book_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "health_avg_liquidations_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "health_avg_open_interest_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "health_avg_trades_BTC-USDT-PERP": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]"
            },
            "perf": {
                "cpu_095_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "cpu_095_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "cpu_absent_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "cpu_absent_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "cpu_avg_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "cpu_avg_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "cpu_max_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "cpu_max_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "cpu_min_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "cpu_min_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "mem_095_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "mem_095_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "mem_absent_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "mem_absent_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "mem_avg_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "mem_avg_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "mem_max_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "mem_max_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('::1', 9090, 0, 0)]",
                "mem_min_data-feed-container": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]",
                "mem_min_redis": "ClientConnectorError: Cannot connect to host localhost:9090 ssl:default [Connect call failed ('127.0.0.1', 9090)]"
            }
        }
    }
}